### DISCUSSION TITLE
Yarn workspaces

### DISCUSSION THREAD

**Discussion by maxfi** [blitz-js/blitz#1969] [Q&A]
Has anyone had any luck using blitz in a yarn workspaces monorepo? With a newly scaffolded blitz app (v0.30.1) I'm getting some file not found errors and have a feeling it might be related to hoisting. I've never used yarn workspaces before so am not sure if this is the case. Would be great if someone could point me in the right direction and I can put together an example. Thanks.

**Comment by flybayer** [comment]
I'm not sure! If you can post some logs and errors, I might can help. A repo reproduction would be nice too

**Reply by maxfi** [reply]
Sorry @flybayer, I should have included a reproduction before. Please see https://github.com/maxfi/blitz-monorepo-example.

Each commit adds a new app resulting in two blitz apps and one vanilla nextjs app. Between each commit I deleted `node_modules` folders, the `yarn.lock` file and reran `yarn install`. It seems that the addition of the vanilla nextjs app breaks dependency resolution.

**Reply by flybayer** [reply]
Well that's an odd one. I tried a few things but couldn't get it to work.

Can you open a full github issue for this? (instead of this discussion)
